glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -170,10 +170,15 @@ open class GleanInternalAPI internal constructor () {
         sendPing("events")
     }
 
-    private fun sendPing(pingName: String) {
-        LibGleanFFI.INSTANCE.glean_send_ping(handle, pingName)
-        // TODO: 1551692 Only do this when ping content was actually queued
-        PingUploadWorker.enqueueWorker()
+    internal fun sendPing(pingName: String) {
+        val queued = LibGleanFFI.INSTANCE.glean_send_ping(
+            handle,
+            pingName,
+            if (configuration.logPings) 1 else 0
```

Comment:
We now have [`Boolean.toByte`](https://github.com/mozilla/glean/blob/ccef0fd863560e4ca3555db7405ffd0528416d15/glean-core/android/src/main/java/mozilla/telemetry/glean/rust/LibGleanFFI.kt#L16) and can use it here.

Reply:
Ah, cool.